config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   get 'static_pages/add_item'
   post 'dish/add_to_cycle'
+  get 'pages/menu'
+  get 'menu', to: 'menu#menu'
+  get 'menu/:date', to: 'menu#new_date'
```

Comment:
Is this route being used?

Reply:
pages/menu is no longer being used but menu and menu/:date are